flutter_app/lib/widgets/display_all_logs_widget.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:imacs/modules/get_airside_logs.dart';
+
+class LogReader extends StatelessWidget{
+
+  const LogReader ({super.key, required this.getAirsideLogs});
+
+  final GetAirsideLogs getAirsideLogs;
+   
+  @override
+  Widget build (BuildContext context){
+    return SizedBox(
+      height: 300,
+      child: AspectRatio(
+        aspectRatio: 16/9,
+        child: ListView.builder(
+          itemCount: getAirsideLogs.getFiles().length,
+          itemBuilder: (context, index){
+            return Padding (
+              padding: const EdgeInsets.all(8.0),
+              child: ElevatedButton (
+                child: Text (getAirsideLogs.getFiles()[index].path),
+                onPressed: () async {
+                  String fileContent = await getAirsideLogs.getFiles()[index].readAsString();
+                  Navigator.push(context, MaterialPageRoute(
+                    builder: (context) => LogDisplayer(fileContext: fileContent),
+                    )
+                  );  
+                } 
+              ),
+            );
+          },
+        ),
+      ),
+    );
+  }
+}
+
+class LogDisplayer extends StatelessWidget{
```

Comment:
Move this to the screens folder and rename it to something like `LogDisplayerScreen`

Reply:
i had tried that, but that widget is dependant on the input from the logreader widget so i'm unable to move it without having the info for its parameters defined in the screens folder